BUILD

```diff
@@ -0,0 +1,131 @@
+repo = ["WORKSPACE",
+         ".bazelrc",
+         "//scala:sources",
+         "//scala/support:sources",
+         "//src/java/com/google/devtools/build/lib:sources",
+         "//src/java/io/bazel/rulesscala/io_utils:sources",
+         "//src/java/io/bazel/rulesscala/jar:sources",
+         "//src/java/io/bazel/rulesscala/scalac:sources",
+         "//src/java/io/bazel/rulesscala/worker:sources",
+         "//src/scala:sources",
+         "//src/scala/io/bazel/rules_scala/scrooge_support:sources",
+         "//src/scala/io/bazel/rules_scala/tut_support:sources",
+         "//src/scala/scripts:sources",
+         "//test:sources",
+         "//test/data:sources",
+         "//test/src/main/resources/scala/test:sources",
+         "//test/src/main/scala/scala/test/srcjars:sources",
+         "//test/src/main/scala/scala/test/twitter_scrooge:sources",
+         "//test/src/main/scala/scala/test/twitter_scrooge/thrift:sources",
+         "//test/src/main/scala/scala/test/twitter_scrooge/thrift/thrift2:sources",
+         "//test/src/main/scala/scala/test/twitter_scrooge/thrift/thrift2/thrift3:sources",
+         "//test/src/main/scala/scala/test/twitter_scrooge/thrift/thrift2/thrift4:sources",
+         "//test/tut:sources",
+         "//test_expect_failure/disappearing_class:sources",
+         "//test_expect_failure/scala_library_suite:sources",
+         "//test_expect_failure/transitive/java_to_scala:sources",
+         "//test_expect_failure/transitive/scala_to_java:sources",
+         "//test_expect_failure/transitive/scala_to_scala:sources",
+         "//thrift:sources",
+         "//tut_rule:sources",
+         "//twitter_scrooge:sources"]
+
+additions = [] 
```

Comment:
Bad naming... and currently redundant. At first I used this mechanism to only run tests that run bazel, the "additions" allowed me to define bazel arguments / flags that will be added to each execution. Later on I saw that shipping .bazelrc is much more effective because some other tests call bash function with inner calls to bazel.